assets/app/js/workbenchDB.js

```diff
@@ -377,7 +380,7 @@ class WorkbenchDB {
   _addFlattenedFiles(files) {
     // Fix for issue #232
     $.each(files, (i, file) => {
-      if (file.type === 'directory' && file.hasOwnProperty('size_count')) {
+      if (file.type === 'directory' && Object.prototype.hasOwnProperty.call(file, 'size_count')) {
```

Comment:
What's the rationale for the change in calling syntax?

Reply:
I got the warning that calling hasOwnProperty directly on object is deprecated. Hence, changed it to the recommended syntax
Reference: https://dev.to/aman_singh/what-s-the-deal-with-object-prototype-hasownproperty-call-4mbj

Reply:
thanks